README.md

```diff
@@ -98,6 +98,11 @@ You can test the SDK by running
 poetry run ci_test
 ```
 
+In local development you can use
+```
+poetry run pytest -m "not file_component_sources"
```

Comment:
Maybe we should create a poetry script for that something like `test.local` or something like that.

Reply:
will do